content/en/docs/appstore/app-services/general-purpose-ocr.md

```diff
@@ -6,39 +6,191 @@ tags: ["Document Service", "AI", "ML", "OCR", "Industrial", "Manufacturing"]
 
 ## 1 Introduction
```

Comment:
@VivekSinghRawat-1  following addition required
1. In the introduction explain how GPO works, introduction very small. You can refer to AWS documentation explaining how the document is divided into multiple groups.
2. Add a section explaining the prepackaged domain model, take an example document and explain which data is stored in which domain model part.